crates/torii/core/src/sql.rs

```diff
@@ -321,52 +320,22 @@ impl Sql {
         block_timestamp: u64,
     ) -> Result<()> {
         let entity_id = format!("{:#x}", entity_id);
+        let model_id = format!("{:#x}", compute_selector_from_tag(&entity.name()));
```

Comment:
we already have the `model_id` in the event data, can we pass it directly instead of computing it like we do for `set_entity`